### ISSUE TITLE
Implement ini config file

### ISSUE THREAD

**Issue by gabri94** [AdvancedNetworkingSystems/prince#4]
A config file reader must be set to load the following parameters:

rp_host  (hostname)
rp_port = (int)
rp_type = (olsr/oonf)
node_id =  (string of main ip / originator)

**Comment by gabri94** [comment]
heuristic = (bool)
weighted_graph =(bool)

**Comment by gabri94** [comment]
implemented in d63007a1f76f6fa14e2a3a6859d183d8324d958

**Comment by gabri94** [comment]
Implemented in:   a7e89fa

**Comment by gabri94** [comment]
Use INI format with this library:   https://github.com/benhoyt/inih

**Comment by gabri94** [comment]
implemented ini reader in 2f0ef0ed2

example:

```
[proto]
protocol = olsr
host  = home.unname.eu
port = 9090
self_id  = 172.19.186.1
refresh = 10

[graph-parser]
heuristic  = 1
weights  = 1
```
